include/detail/gpu_rnnt_kernel.h

```diff
@@ -163,17 +163,64 @@ __global__ void compute_grad_kernel(Tp* grads, const Tp* const acts, const Tp* c
             Tp grad = exp(alphas[col] + betas[col] + logpk - logll[mb]);
             // grad to last blank transition
             if (idx == blank_ && t == T-1 && u == U-1) {
-              grad -= exp(alphas[col] + logpk - logll[mb]);
+                grad -= exp(alphas[col] + logpk - logll[mb]);
             }
             if (idx == blank_ && t < T-1) {
-              grad -= exp(alphas[col] + logpk - logll[mb] + betas[col + maxU]);
+                grad -= exp(alphas[col] + logpk - logll[mb] + betas[col + maxU]);
             }
             if (u < U-1 && idx == labels[u]) {
-              grad -= (1. + fastemit_lambda_) * exp(alphas[col] + logpk - logll[mb] + betas[col+1]);
+                grad -= exp(alphas[col] + logpk - logll[mb] + betas[col+1]);
             }
             grads[col * alphabet_size + idx] = grad;
 
             idx += NT;
         }
     }
 }
+
+template<int NT, typename Tp>
+__global__ void compute_fastemit_grad_kernel(Tp* grads, const Tp* const acts, const Tp* const denom, const Tp* alphas, const Tp* betas, const Tp* const logll, const int* const xlen, const int* const ylen, 
+    const int* const mlabels, const int minibatch, const int maxT, const int maxU, const int alphabet_size, const int blank_, const Tp lambda) {
```

Comment:
Nitpick: Could you use `fastemit_lambda` instead of lambda, please? Just to avoid any confusion :)

Reply:
Sure! I submitted a commit. Is everything okay?